admin/templates/configuration/etc/log4cxx.unittest.properties

```diff
@@ -0,0 +1,16 @@
+#
```

Comment:
Having examples is OK, but I don't think we need to install examples in run directory. `etc` directory only needs to have stuff which is useful for production, not debugging. Keeping it in the source but not installing is OK, or maybe we can have separate directory for examples.